fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java

```diff
@@ -75,6 +77,46 @@ public void run() {
             }
         }
         LOG.info("Internal schema is initialized");
+        Optional<Database> op
+                = Env.getCurrentEnv().getInternalCatalog().getDb(StatisticConstants.DB_NAME);
+        if (!op.isPresent()) {
+            LOG.warn("Internal DB got deleted!");
+            return;
+        }
+        Database database = op.get();
+        modifyTblReplicaCount(database, StatisticConstants.ANALYSIS_TBL_NAME);
+        modifyTblReplicaCount(database, StatisticConstants.STATISTIC_TBL_NAME);
+        modifyTblReplicaCount(database, StatisticConstants.HISTOGRAM_TBL_NAME);
+    }
+
+    public void modifyTblReplicaCount(Database database, String tblName) {
+        if (!(Config.min_replication_num_per_tablet < 3 && Config.max_replication_num_per_tablet >= 3)) {
+            return;
+        }
+        while (true) {
+            if (Env.getCurrentSystemInfo().aliveBECount() >= 3) {
```

Comment:
check tables current replica num, if all tables' replica num is 3 then exit this thread